src/protagonist/API/Features/Image/Ingest/DeliveryChannelProcessor.cs

```diff
@@ -0,0 +1,209 @@
+﻿using System.Collections.Generic;
+using API.Exceptions;
+using DLCS.Core.Collections;
+using DLCS.Model.Assets;
+using DLCS.Model.DeliveryChannels;
+using DLCS.Model.Policies;
+using Microsoft.Extensions.Logging;
+
+namespace API.Features.Image.Ingest;
+
+public class DeliveryChannelProcessor
+{
+    private readonly IDefaultDeliveryChannelRepository defaultDeliveryChannelRepository;
+    private readonly IDeliveryChannelPolicyRepository deliveryChannelPolicyRepository;
+    private readonly ILogger<DeliveryChannelProcessor> logger;
+    private const string None = "none";

```

Comment:
would this be better as `NoneChannel`? could possibly get confused as meaning the policy with the same name

Reply:
Updated name, this is policy rather than channel (which highlights the point you made perfectly!) so named `NonePolicy`